mkt/search/tests/test_views.py

```diff
@@ -1507,12 +1528,13 @@ def test_search_popularity(self):
         eq_(objs[2]['slug'], self.webapp.app_slug)
 
     def test_search_sort_by_reviewed(self):
+        self.makeHomescreen()
```

Comment:
same as above, a separate test is probably better, those tests depend on relevancy/reviewed time, so they are quite fragile. your call though.